recipes/mg5amcnlo/meta.yaml

```diff
@@ -0,0 +1,160 @@
+{% set name = "mg5amcnlo" %}
+{% set version = "3.5.7" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/hep-packaging-coordination/mg5amcnlo-release-mirror/releases/download/{{ version }}/MG5_aMC_v{{ version }}.tar.gz
+    sha256: ca3e027f078438318bd4143a102aebc2d51358f537b5d52bbc574fe49a6a901e
+    folder: source
+    patches:
+      # FIXME: mg5amcnlo assumes that ninja and collier are only static libraries
+      - allow-for-shared-libraries.patch
+
+build:
+  # FIXME: mg5amcnlo doesn't support Python 3.12 yet
+  skip: true  # [py<37 or py>311 or win]
+  number: 0
+  script:
+    # Remove unnecessary files
+    - rm -r source/.github
+    - rm -r source/tests
+    - rm -r source/vendor/CutTools
+    - rm source/.bzrignore
+    - rm source/.gitignore
+    - rm source/code-notes
+    - rm source/README
+    - rm source/UpdateNotes.txt
+
+    # Fix bugs
+    # FIXME: Use user space perl instead of system perl
+    - find source -type f -exec grep -Il '#!/usr/bin/perl -w' {} + | xargs sed -i 's|#!/usr/bin/perl -w|#!/usr/bin/env perl -w|g'
+
+    - cp -r source $PREFIX/MG5_aMC
+
+    # TODO: Add MG5aMC_PY8_interface
+    # https://github.com/mg5amcnlo/MG5aMC_PY8_interface
+
+    # Build configuration file be replacing placeholder values with PREFIX
+    - cp $RECIPE_DIR/mg5_configuration.txt .
+    - sed -i "s|REPLACE_WITH_PREFIX|$PREFIX|g" mg5_configuration.txt
+    - sed -i "s|REPLACE_WITH_BASENAME_FC|$(basename $FC)|g" mg5_configuration.txt
+    - sed -i "s|REPLACE_WITH_BASENAME_GXX|$(basename $GXX)|g" mg5_configuration.txt
+    - mv mg5_configuration.txt $PREFIX/MG5_aMC/input/mg5_configuration.txt
```

Comment:
`mg5amcnlo` looks up all the tools it knows about and can use for code generation using `MG5_aMC/input/mg5_configuration.txt`. The author assumed you would always compile everything from scratch on every machine you ever used it on. To avoid this and get the correct paths for the conda-forge packages installed as `run` requirements, use `REPLACE_WITH_` variables in `mg5_configuration.txt` to overwrite with the value of their `$PREFIX` paths so they will be found correctly at install time.

Instead of trying to properly `sed -i` edit each line of `mg5_configuration.txt` during the build, include `mg5_configuration.txt` as a file in the feedstock source to then overwrite and `mv` into what is packaged.